app/views/maths_and_physics/claims/_ineligibility_reason_ineligible_current_school.html.erb

```diff
@@ -5,3 +5,10 @@
 <p class="govuk-body">
   You can only get this payment if you are employed to teach at an eligible state-funded secondary school.
 </p>
+
+<p class="govuk-body">
+  For more information, including regional and employment eligibility criteria for this payment,
+  visit the
+  <%= link_to "’Claim a payment for teaching maths or physics’", "https://www.gov.uk/government/publications/additional-payments-for-teaching-eligibility-and-payment-details/claim-a-payment-for-teaching-maths-or-physics-eligibility-and-payment-details#eligible-subjects", class: "govuk-link" %>
```

Comment:
Could we use `MathsAndPhysics.eligibility_page_url` and then pop the fragment on the end here? I.e.:

```erb
   <%= link_to "’Claim a payment for teaching maths or physics’", "#{MathsAndPhysics.eligibility_page_url}#eligible-subjects", class: "govuk-link" %>
```

Reply:
Nice! That url was way too long! Thanks!